app/page.tsx

```diff
@@ -1,75 +1,96 @@
 'use client';
 
 import React from 'react';
-import { useForm, Controller } from 'react-hook-form';
-import { Select, MenuItem, TextField, FormControl, InputLabel, Button, Box, Checkbox, FormControlLabel } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import InputText from '@/components/InputText';
 
 type FormValues = {
   firstName: string;
   lastName: string;
-  email: string;
-  phone: string;
-  whiskey: string;
-  quantity: number;
   address: string;
-  ageVerified: boolean;
+  phoneNumber: string;
 };
 
-export default function WhiskeyPurchaseForm() {
-  const { register, handleSubmit, control } = useForm<FormValues>({
-    defaultValues: {
-      firstName: '',
-      lastName: '',
-      email: '',
-      phone: '',
-      whiskey: 'Whiskey Option 1',
-      quantity: 1,
-      address: '',
-      ageVerified: false,
-    },
-  });
-
-  const whiskeyOptions = ['Whiskey Option 1', 'Whiskey Option 2', 'Whiskey Option 3'];
+export default function Page() {
```

Comment:
Do you have `ESLint` working? This type of component declaration should not be allowed.

Reply:
<img width="829" alt="image" src="https://github.com/javiercodixis/whiskeyjs/assets/143386814/756765d2-1984-4c6f-82d5-6898a2eed17a">
About this I have the arrow function definition, right? but for some reason is not creating a warning